### PR TITLE
addes page for managing metrics account.

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages to review

- https://deploy-preview-443--logz-docs.netlify.com/user-guide/infrastructure-monitoring/metrics-account.html

### CODE REVIEW

File: _source/_data/toc.yml [hunk 1]
```diff
@@ -71,9 +71,9 @@ firstLevel:
   - title: Infrastructure Monitoring
     url: /user-guide/infrastructure-monitoring/
     thirdLevel:
-    - title: Data rollups
-      url: /user-guide/infrastructure-monitoring/data-rollups.html
-    - title: Configure Grafana drilldown links
```
Comment (shalper) [inline _source/_data/toc.yml]: @imnotashrimp  - This slipped into the master by mistake. Needs to be erased here and added from the right PR.

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+If the Metrics tab is missing, the account doesn't have permissions to it. You can give it permissions if you're an admin, as shown next.
+
+
+#### To add view access to your Metrics account
+
+You can give log-monitoring sub-accounts access to view Metrics accounts that belong to the same main account.
+
+Select [**<i class="li li-gear"></i> > Settings > Manage accounts**](https://app.logz.io/#/dashboard/settings/manage-accounts) from the top-right corner. (This option only appears when you are on the main account.)
+
+Scroll down to the Metrics account, and select it to expand its panel. Here you'll find the account's shipping token, account name, and usage history.
```
Comment (shalper) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]:
```suggestion
Scroll down to the Metrics account, and select it to expand its panel. Here you'll find the account's shipping token, account name, and usage history. 
```

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#### To add view access to your Metrics account
+
+You can give log-monitoring sub-accounts access to view Metrics accounts that belong to the same main account.
+
+Select [**<i class="li li-gear"></i> > Settings > Manage accounts**](https://app.logz.io/#/dashboard/settings/manage-accounts) from the top-right corner. (This option only appears when you are on the main account.)
+
+Scroll down to the Metrics account, and select it to expand its panel. Here you'll find the account's shipping token, account name, and usage history. 
+You can add any of the log-monitoring sub-accounts in the drop-down list. If the account you are looking for doesn't appear on the list, it likely belongs to a different main account.
+
+![Managing Metrics account permissions](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/metrics-account-read-permissions.png)
```
Comment (shalper) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]: @imnotashrimp  - This image was updated in AWS but doesn't seem to be updated. https://logzbucket.s3-eu-west-1.amazonaws.com/logz-docs/grafana/metrics-account-read-permissions.png. What am I missing?

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+[Metrics tab](https://app.logz.io/#/dashboard/grafana/).
+
+![Navigate to your metrics account](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/reach-metrics.png)
+
+If the Metrics tab is missing, the account doesn't have permissions to it. You can give it permissions if you're an admin, as shown next.
+
+
+#### To add view access to your Metrics account
+
+You can give log-monitoring sub-accounts access to view Metrics accounts that belong to the same main account.
```
Comment (imnotashrimp) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]:
```suggestion
You can give log-monitoring sub accounts access to view Metrics accounts that belong to the same main account.
```

`sub accounts` is never hyphenated

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+permalink: /user-guide/infrastructure-monitoring/metrics-account.html
+flags:
+  admin: true
+  logzio-plan:
+tags:
+contributors:
+  - shalper
+---
+
+By default, your Metrics account is found under your main log monitoring account.
```
Comment (imnotashrimp) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]:
```suggestion
By default, your Metrics account can be accessed from only your main account.
```

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+To open it, log into your Logz.io Operations account and select the
```
Comment (imnotashrimp) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]: 1. `To open it` - not clear if `it` refers  to your logging or your metrics account
2. No such thing as "Operations account" - change to `Operations workspace`

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+[Metrics tab](https://app.logz.io/#/dashboard/grafana/).
+
+![Navigate to your metrics account](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/reach-metrics.png)
```
Comment (imnotashrimp) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]: There's a lot going on in this image - I recommend cropping it in to just the areas you need

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+
+#### To add view access to your Metrics account
+
+You can give log-monitoring sub-accounts access to view Metrics accounts that belong to the same main account.
+
+Select [**<i class="li li-gear"></i> > Settings > Manage accounts**](https://app.logz.io/#/dashboard/settings/manage-accounts) from the top-right corner. (This option only appears when you are on the main account.)
+
+Scroll down to the Metrics account, and select it to expand its panel. Here you'll find the account's shipping token, account name, and usage history. 
+You can add any of the log-monitoring sub-accounts in the drop-down list. If the account you are looking for doesn't appear on the list, it likely belongs to a different main account.
```
Comment (imnotashrimp) [inline _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]: same notes here:

`log-monitoring sub-accounts` => `sub accounts`

File: _source/user-guide/infrastructure-monitoring/managing-metrics-account.md (hunk 3 above)
Comment (imnotashrimp) [reply _source/user-guide/infrastructure-monitoring/managing-metrics-account.md]: Not sure. But there are some things on this image that need to change - 

1. Blur the shipping token & account id
2. remove the hand cursor

**Comment by imnotashrimp** [comment]
@shalper Taking another look at the docs - There's already a page for managing the Infrastructure Monitoring account. This page can be merged with that one, and they should live under the "Accounts" section in the TOC.

**Pull request closed without merging**